playwright/e2e/login/login.spec.ts

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { checkA11y, injectAxe } from "axe-playwright";
+
+import { test, expect } from "../../element-web-test";
+import { HomeserverInstance } from "../../plugins/utils/homeserver";
+import { startHomeserver, stopHomeserver } from "../../plugins/homeserver";
+
+test.describe("Consent", () => {
+    let homeserver: HomeserverInstance;
+
+    test.afterEach(() => {
+        stopHomeserver(homeserver);
+    });
+
+    test.describe("m.login.password", () => {
+        const username = "user1234";
+        const password = "p4s5W0rD";
+
+        test.beforeEach(async ({ page, app }) => {
+            homeserver = await startHomeserver("consent");
+
+            await app.registerUser(homeserver, username, password);
+            await page.goto("/#/login");
+        });
+
+        test("logs in with an existing account and lands on the home screen", async ({ page }) => {
```

Comment:
Since we decided to not have the same tests running in both cypress and playwright, can you disable this test in the cypress suite?

Reply:
Sure